ansible_collections/arista/avd/plugins/action/anta_workflow.py

```diff
@@ -107,12 +107,23 @@
             "md_output": {"type": "str"},
             "json_output": {"type": "str"},
             "filters": {
+                "type": "dict",
+                "options": {"exclude_statuses": {"type": "list", "elements": "str", "choices": ["success", "failure", "error", "skipped", "unset"]}},
```

Comment:
Alphabetical order please for choices. You need to update the `status_priority` choices as well.
```suggestion
                "options": {"exclude_statuses": {"type": "list", "elements": "str", "choices": ["success", "failure", "error", "skipped", "unset"]}},
```

Reply:
Updated.